tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -1140,34 +1136,33 @@ def test_positive_end_to_end(self, fake_manifest_is_set, default_sat, rhel6_cont
         repositories = []
 
         # step 2.5: Create custom YUM repository
-        repo1 = entities.Repository(
+        yum_repo = entities.Repository(
             server_config, product=prod, content_type='yum', url=CUSTOM_RPM_REPO
         ).create()
-        repositories.append(repo1)
+        repositories.append(yum_repo)
 
         # step 2.6: Enable a Red Hat repository
         if fake_manifest_is_set:
-            repo3 = entities.Repository(
+            rhel_repo = entities.Repository(
                 id=enable_rhrepo_and_fetchid(
                     basearch='x86_64',
                     org_id=org.id,
-                    product=PRDS['rhel'],
-                    repo=REPOS['rhva6']['name'],
-                    reposet=REPOSET['rhva6'],
-                    releasever='6Server',
+                    product=constants.PRDS['rhel'],
+                    repo=constants.REPOS['rhst7']['name'],
+                    reposet=constants.REPOSET['rhst7'],
+                    releasever=None,
```

Comment:
Instead update `releasever` in `enable_rhrepo_and_fetchid` to set default value to None!

Reply:
Updated